### PR TITLE
[INTEL oneDNN] Refactoring: replace/remove MKL ML API calls

### PR DESCRIPTION
**Pull request by gzmkl**
MKL ML has been replaced by MKL DNN and now oneDNN. It is not supported anymore. 
In TensorFlow, there are still some MKL calls even though related methods and tests are NOT called any more. 

The PR replaces and removes calls to MKL ML (cblas) APIs which are not supported long time ago.

1. For FP32 Matmul, oneDNN dnnl_gemm() will be called.
2. For FP64 Matmul, it will NOT be supported as oneDNN currently does not support it. 

NEXT STEP: clean up MKL ML related configuration (bazel files).

### CODE REVIEW

**Comment by gzmkl** [comment]
@penpornk Hi Penporn, I noticed that there were failures related internal CI and AMD ROCm build. But I am not sure if this PR caused them. The old should NOT be executed in recent years after we moved from MKL ML to MKL DNN (around 2017 or 18). 
Please help to check. Thanks!

My next step is to clean up related bazel configuration (to get rid of "if_mkl_ml" staff).          -GZ

File: tensorflow/compiler/xla/service/cpu/runtime_matmul_mkl.h [hunk 1]
```diff
@@ -19,24 +19,16 @@ limitations under the License.
 #include <iostream>
 
 #ifdef ENABLE_MKL
-#include "third_party/intel_mkl_ml/include/mkl_cblas.h"
+#include "dnnl.hpp"
```
Comment (penpornk) [inline tensorflow/compiler/xla/service/cpu/runtime_matmul_mkl.h:22]: Could you please use the full path to the header file from now on? Thank you!

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR! I will hold off on merging this PR until after TF 2.13 branch cut.

**Comment by penpornk** [comment]
@gzmkl I haven't seen the latest ROCm results on this PR. (It's still running at the time I'm writing this comment.) But ROCm has been having infra failure lately. (It failed in many other PRs.) So it's likely unrelated to this PR. I'll take a look again after it's done running. :)

File: tensorflow/compiler/xla/service/cpu/runtime_matmul_mkl.h (hunk 1 above)
Comment (gzmkl) [reply tensorflow/compiler/xla/service/cpu/runtime_matmul_mkl.h:22]: Sure, I will make the change. Thanks

**Comment by gzmkl** [comment]
@penpornk  Please help to check the latest ROCm results on this PR. I could not find anything with the "Details" link.  THANKS

**Comment by gzmkl** [comment]
This PR is replaced by the new one (hopefully no more conflicts after oneDNN 3.x integration)

https://github.com/tensorflow/tensorflow/pull/61471

Thus close this one

**Pull request closed without merging**